tilelang/tools/plot_layout.py

```
@@ -212,3 +213,86 @@ def plot_layout(
+            dst_idx = frag.map_forward_index(item)[0].value
+            name = "(" + ",".join([str(i) for i in apply_idx_fn(*item)]) + ")"
+            src_flat_idx[dst_idx, th] = src_idx
+            src_idx_str[dst_idx, th] = name
+
+    plt.figure(figsize=(item_scale * num_thread_dim, item_scale * num_local_dim))
+    cmap = plt.get_cmap(colormap)
+    plt.pcolormesh(src_flat_idx, cmap=colormap, edgecolors="k", linewidth=0.5)
+    mx = np.max(src_flat_idx)
+    for i in range(num_local_dim):
+        for j in range(num_thread_dim):
+            r, g, b, a = cmap(src_flat_idx[i, j] / mx)
```

Ruff (0.14.14)

[warning] Unpacked variable `a` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)